packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -278,30 +279,33 @@ function filterSettingsToParams(filterSettings: FilterSettings): any {
   }
   
   const tagsSoftFiltered = _.filter(filterSettings.tags, t => (t.filterMode!=="Hidden" && t.filterMode!=="Required" && t.filterMode!=="Default" && t.filterMode!==0));
-  let scoreExpr: any = null;
-  if (tagsSoftFiltered.length > 0 || frontpageSoftFilter.length > 0) {
```

Comment:
Reviewers, notice:
Removing this `if` is supposed to fix an existing bug in the codebase:

The bug is:
If `tagsSoftFiltered` and `frontpageSoftFilter` are both empty,
then `defaultScoreModifiers` will be ignored